INTITA/JSONFiles/BaseProtocol.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  BaseProtocol.swift
+//  INTITA
+//
+//  Created by Anastasiia Spiridonova on 12.02.2021.
+//
+
+import Foundation
+
+protocol BaseProtocol: Codable {
```

Comment:
I suggest you rename the common term "Base" on "Location" everywhere. Also the common term "values" could be named "locations"